lib/widgets/select_wallet_box.dart

```diff
@@ -59,12 +56,23 @@ class SelectWalletBox extends StatelessWidget {
         ),
         margin: EdgeInsets.all(8),
         child: Row(children: [
+          SizedBox(width: 8),
           Container(
-            color: extendedTheme.selectedTextColor,
-            width: 30,
-            height: 30,
-            child: Identicon(seed: walletId, size: 8),
-          ),
+              decoration: BoxDecoration(
+                  color: WitnetPallet.black,
+                  border: Border.all(color: WitnetPallet.transparent),
+                  borderRadius: BorderRadius.all(Radius.circular(24))),
+              child: ClipRRect(
+                borderRadius: BorderRadius.all(Radius.circular(24)),
+                child: Container(
+                  decoration: BoxDecoration(
+                      color: WitnetPallet.black,
+                      border: Border.all(color: WitnetPallet.black)),
+                  width: 30,
+                  height: 30,
+                  child: Identicon(seed: walletId, size: 8),
+                ),
+              )),
           Expanded(
```

Comment:
The new font family makes the wallet item text look too big. Is it possible to make it smaller?
![Screenshot 2024-10-07 at 16 17 11](https://github.com/user-attachments/assets/6aad3202-1579-46cb-a201-b65104b72e29)